ui/public/about.html

```diff
@@ -15,29 +16,36 @@
             color: black;
             font-family: monospace;
         }
+        
         main {
             text-align: center;
             margin-top: 3em;
         }
+        
         p {
             margin: 0.25em;
             font-size: 0.85em;
         }
+        
         button {
             margin: 1em;
             width: 6em;
         }
     </style>
 </head>
+
 <body>
     <main>
         <img src="Square89x89Logo.png" />
         <h3>Manta Signer</h3>
-        <p>Version 0.4.1</p>
+        <p>Version 0.5.0</p>
         <p>Copyright © 2019-2021 Manta Network</p>
-        <p><a href="https://github.com/manta-network/manta-signer">manta-network/manta-signer</a></p>
-        <br>
-        <p style="font-size: 0.75em; color: red">DOLPHIN TESTNET</p>
+        <p>
+            <a href="https://github.com/manta-network/manta-signer">manta-network/manta-signer</a
+        >
```

Comment:
Can we avoid the line break that splits open the tag? Is this an auto-formatting?

Reply:
Yeah, auto-formatting, how annoying